hack/local-up-operator.sh

```diff
@@ -17,176 +17,64 @@
 # This command runs tidb-operator in Kubernetes.
 #
 
-set -o errexit
-set -o nounset
-set -o pipefail
-
-ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
-cd $ROOT
-
-source "${ROOT}/hack/lib.sh"
+# Default provider is kind
+PROVIDER=${PROVIDER:-kind}
 
+# Function to display usage information
 function usage() {
     cat <<'EOF'
-This commands run tidb-operator in Kubernetes.
+This script runs tidb-operator in Kubernetes using the appropriate provider.
 
-Usage: hack/local-up-operator.sh [-hd]
+Usage: router.sh [-hd] [-p PROVIDER]
```

Comment:
Do you want to rename this file to `router.sh`? or just use `hack/local-up-operator.sh` instead?

Reply:
I overlooked this.
Maybe `local-up-operator.sh` is a better name, it could keep the reference consistent, if someone do referenced it.